_includes/working-with-tables.md

```diff
@@ -0,0 +1,208 @@
+1. Copy the items from the table below (do not copy the "Sample Type" cell).
+1. Click in the first row of the _Sample Type_ column, and use the key combination to paste.
+
+| Sample Type |
+|-------------|
+|GENOMIC|
+|GENOMIC|
+|GENOMIC|
+|GENOMIC|
+|GENOMIC|
+
+### Fill many
+Entering the same data in all rows can be done by using the "fill from once cell" feature. Enter the data in one cell, then you can fill in multiple cells by:
+  * double-clicking the square in the bottom right corner of that cell to fill in all cells below.
+  * clicking the square in the bottom right corner of that cell, then dragging the mouse up or down the selected number of rows to fill those cells with the initial value.
+
+<a name="fill-down-drag"/>
+
+#### Drag (fills up or down)
+1. Type **10XT** in the _last_ row of the _Project_ column. 
+1. Click on the "10XT" project short name in the dropdown menu.
+    * The cell will have a blue border with a blue square in the bottom right corner. (If you do not see the blue square, click on another cell and then click again on the cell that contains your project short name.)
+1. Click on the blue square and hold the mouse button down as you drag up to the top row in the _Project_ column. Release the mouse button.
+    * The cells will all change to contain the 10XT project short name.
+
+<a name="fill-down-click"/>
+
+#### Double-click (fills down)
+1. Type your project's short name in the _first_ row of the _Project_ column.
+1. Click on your project in the dropdown menu.
+    * The cell will have a blue border with a blue square in the bottom right corner. (If you do not see the blue square, click on another cell and then click again on the cell that contains your project short name.)
+1. Double-click on the blue square in the bottom right corner of the cell that contains your project short name.
+    * The cells below it will all change to contain your project short name.
+
+*Note*: multiple columns can be filled down at once. 
+{% if include.detailed == true %}
+You can test this out:
+1. Fill in values in the top row for the _Tissue Origin_, _Tissue Type_, _Times Received_, and _Tube Number_ columns.
+1. Click the _Tissue Origin_ cell, then press **Shift**, click the _Tube Number_ cell, and release the Shift key.
+    * All the cells between _Tissue Origin_ and _Tube Number_ will be selected. 1. Double-click or drag down on the blue square in the bottom right corner of the _Tube Number_ cell.
+    * All the cells in all the selected columns will change to contain the values from the first row.
+{% endif %}
+
+<a name="fill-down-increment"/>
+
+### Incrementing fill down
+If the values in a column need to each be one greater than the value in the row before it, you can fill the column using the incremental fill down.
+
+1. In the first two rows of the _Matrix Barcode_ column, enter the following (substituting _XX_ for your initials):
+
+|Matrix Barcode|
```

Comment:
Table breaks numbering. Needs to be indented a few spaces (4 maybe?) to prevent that